Source/Scene/SkyAtmosphere.js

```diff
@@ -133,6 +139,13 @@ define([
     /**
      * @private
      */
+    SkyAtmosphere.prototype.setDayNight = function(enableLighting) {
```

Comment:
This is private, but it was added to CHANGES.md.  It is fine for it to be private, but CHANGES.md should not mention it.